managed-services/iot-hub/reference-content/certificate.mdx

```diff
@@ -0,0 +1,67 @@
+content:
+  h1: Certificate Authority update
+  paragraph: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+dates:
+  validation: 2023-04-27
+  posted: 2023-04-27
+categories:
+  - managed-services
+tags: iot iot-hub ca certificate authority update
+---
+
+As a security measure, Scaleway IoT Hub was designed with a mutual Transport Layer Security (TLS) as the default authentication protocol. This means each device needs both a certificate and key pair to connect to IoT Hub.
+
+Using a trusted Certificate Authority (CA) to issue digital certificates is essential to guarantee a secure Transport Layer Security (TLS) connection between a device and the IoT Hub. A digital certificate contains important information about the server's identify that can help verify its authenticity, then allowing the device to use it to establish a secure connection with the server.
+
+However, the current Certificate Authority of the IoT Hub server endpoint is set to expire **in February 2024**.
+
+The information provided on this page explains what the migration entails and how to successfully transition to the new CA.
+
+<Message type="note">
+  It is possible to replace the Scaleway-generated default certificate with either a self-generated or third-party certificate. If this applies to you and your devices do not use TLS, the information contained within this page will not pertain to you.
+</Message>
+
+The CA is a `.pem` bundle that currently contains both the soon-to-be expired and new CA. This is part of our strategy to make the transition seamless to you.
+
+# Planning the migration
+
+The reason for the update is to better your IoT Hub experience by providing you with a new CA that will be valid for 10 years. That way, you devices won't need to be regularly updated. Additionally, the key used for the current CA has been changed from an RSA4096 to an ECDSA key with a NIST P-384 Curve. The new ECDSA algorithm uses shorter key lenghts than RSA for much stronger security, which is ideal embedded devices.
+
+To implement those changes, a new IoT Hub endpoint has been made available: `migration.iot.fr-par-scw.cloud`. It acts exactly like `iot.fr-par-scw.cloud`, which is the endpoint Scaleway users have habitually been provided with. Starting now, you can connect to this new, transitory endpoint.
+
+On **26 September 2023**, the old certificate for the `iot.fr-par-scw.cloud` endpoint will be changed for one signed by the new CA.
+
+On **30 January 2024**, the migration endpoint will be deactivated. You will only be able to connect to the main `iot.fr-par-scw.cloud` endpoint.
+
+Taking this timeline into account and depending on what your device allows, two options are available to you.
+
+## Your device can use a CA bundle 
+
+The first option is to use a CA bundle. With that option, the transition will be smoother because no further action will be required from you once you are done configuring it.
```

Comment:
```suggestion
The first option is to use a CA bundle. In this case, the transition will be smoother because no further action will be required from you once you are done configuring it.
```